composites/Plugin/SnippetEditor/components/SnippetEditor.js

```diff
@@ -528,6 +531,10 @@ SnippetEditor.defaultProps = {
 	mapDataToPreview: null,
 	generatedDescription: "",
 	locale: "en",
+	excludeReplaceVars: [
```

Comment:
In this case, setting a defaultProp that is filled with values has the unintuitive consequence that one needs to explicitly pass `excludeReplaceVars` as an empty array to receive all replaceVars, rather than omitting the prop.

I guess the alternative (having to pass these 8 values in a string almost everytime) is not desirable either though... Can't we set the values to be excluded in each contect in a look-up helper object somewhere? For example
```{
    edit-post: {
        context: "edit-post",
        excludeReplaceVars: [ "your_stuff, "my_stuff", ]
    },
    edit-archive: {
        context: "edit-archive",
        excludeReplaceVars: [ "your_archive_stuff ],
    },
}
```
Etc.? Can't oversee right now whether that would be desirable.

Reply:
After the discussion we had, and with the new instances of the replacement variable editor having a separate architecture, we will keep the excluded variables in the default props. The look-up helper object is a really nice idea though, and is something to keep in mind.